site/src/components/Workspace/Workspace.tsx

```diff
@@ -106,21 +107,13 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
   const { t } = useTranslation("workspacePage")
 
   const buildError = Boolean(workspaceErrors[WorkspaceErrors.BUILD_ERROR]) && (
-    <AlertBanner
-      severity="error"
-      error={workspaceErrors[WorkspaceErrors.BUILD_ERROR]}
-      dismissible
-    />
+    <ErrorAlert error={workspaceErrors[WorkspaceErrors.BUILD_ERROR]} />
```

Comment:
The errors on this page used to be dismissible but now they aren't - do we need to add a prop?

Reply:
Yes. I missed that, sorry! I will check the files again just to be sure I didn't miss any `dismissible` prop

Reply:
Fixed.